src/OctoshiftCLI.Tests/ado2gh/Commands/DownloadLogsCommandTests.cs

```diff
@@ -155,6 +157,62 @@ public async Task Calls_Download_With_Expected_Migration_Log_File()
             mockHttpDownloadService.Verify(m => m.Download(It.IsAny<string>(), migrationLogFile));
         }
 
+        [Fact]
+        public async Task Waits_For_URL_To_Populate_When_Wait_Requested()
+        {
+            // Arrange
+            var githubOrg = "FooOrg";
+            var repo = "foo-repo";
+            var logUrlEmpty = "";
+            var logUrlPopulated = "some-url";
+            var defaultFileName = $"migration-log-{githubOrg}-{repo}.log";
```

Comment:
would you make these `const`s too. Also regarding `defaultFilename` since it's an interpolated string, not sure if CodeQL supports it yet, C# def. supports making an interpolated string `const` (if possible) now but CodeQL is a bit behind the latest C# lang version so it might complain. I guess you can make the change and see if it does or not 😃

Reply:
Updated in 93b135ba09be640054c604d89a7591bf7e40f673!